cf-nodejs-logging-support-core/log-core.js

```diff
@@ -716,6 +736,37 @@ var overrideField = function (field, value) {
     return false;
 };
 
+//sets
+var overrideCustomFieldFormat = function(value) {
+    if(typeof value == "string") {
+        switch (value) {
+            case "cf":
+            case "cloudfoundry":
+            case "application-logging":
+                defaultCustomEnabled = false;
+                cfCustomEnabled = true;
+                break;
+            case "all":
+                defaultCustomEnabled = true;
+                cfCustomEnabled = true;
+                break;
+            case "disabled":
+                defaultCustomEnabled = false;
+                cfCustomEnabled = false;
+                break;
+            case "default":
+            case "cloud-logging":
+            default:
+                defaultCustomEnabled = true;
+                cfCustomEnabled = false;
+        }
```

Comment:
There is a `return` statement in the `else` block. Why is there none at the end of the `if` block?

Reply:
this is an oversight, thanks